2-Regression/1-Tools/translations/assignment.az.md

```diff
@@ -0,0 +1,12 @@
+# Scikit-learn ilə reqressiya
+
+## Təlimatlar
+
+Scikit-learn-dəki [Linnerud data massivinə](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_linnerud.html#sklearn.datasets.load_linnerud) nəzər salın. Bu data massivinin bir neçə [hədəfi](https://scikit-learn.org/stable/datasets/toy_dataset.html#linnerrud-dataset) mövcuddur: 'O fitnes klubundakı 20 orta yaşlı kişidən toplanmış 3 tapşırıq(data) və 3 psixoloji(hədəf) dəyişəndən ibarətdir'.
+
+Öz sözlərinizlə bel xətti ilə oturub-durma hərəkətinin edilmə sayı arasındakı əlaqəni çəkəcək reqressiya modelinin necə yaradılmasını təsvir edin. Data massivindəki digər data nöqtələri üçün də eyni şeyi edin.
+
+## Qiymətləndirmə
+| Kriteriya                       | Örnək                           | Adekvat                      | İnkişaf Etdirilməlidir          |
```

Comment:
Please refer the 2nd class assignment table for the translation. The first header position should be updated too